apps/website-25/src/components/courses/MarkdownExtendedRenderer.tsx

```diff
@@ -2,12 +2,20 @@ import React, { useEffect } from 'react';
 import { evaluate } from '@mdx-js/mdx';
 import { MDXContent } from 'mdx/types';
 import Greeting from './Greeting';
+import Embed from './Embed';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Also I realised I managed to get Callout working without a dependency cycle, but then didn't update the comment in MarkdownExtendedRenderer. So there isn't a circular dependency any more :tada: (but things are set up so that it can be stable if this is needed in future)